app/styles/_variables.scss

```diff
@@ -63,3 +63,8 @@ $fox-logo-zindex: 999;
 //IFrame close button
 $close-button-color: #c5cdd4;
 $close-button-zindex: $fox-logo-zindex + 1;
+
+$image-url-path: "/images/" !default;
+@function image-url($url) {
+  @return url("#{$image-url-path}#{$url}");
+}
```

Comment:
Does this mean that we can remove the following code?
```js
  grunt.config('sass', {
    options: {
      imagePath: '/images'
    },
```
&mdash; via [/grunttasks/sass.js#L7-L9](https://github.com/mozilla/fxa-content-server/blob/b8db477f56abfef7ba6380f835c6d401c9f78fe9/grunttasks/sass.js#L7-L9)